blockchain/easycleos.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
suggest putting the functions all in one file. easier for Docker + there arent so many lines